xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingService.java

```diff
@@ -67,5 +68,8 @@ public ProductSubscription getProductSubscription() {
         return productSubscription;
     }
 
-
+    @Override
+    protected WebSocketClientExtensionHandler getWebSocketClientExtensionHandler() {
+        return null;
```

Comment:
Should this be `BinanceWebSocketHandler.INSTANCE`?  Just based on https://github.com/sanderdg/xchange-stream/commit/0e2dd0866f09f8a6cecc0531ca84e264523c81b8 which is the Bitmex version of this fix

Reply:
This seems to be the case

Reply:
So is this a mistake in the pull request?

Is the pull request actually needed for Binance?

It does sound like it's needed for Bitmex...